src/symbolic/symbolic_choice.ml

```diff
@@ -484,18 +495,22 @@ module Make (Thread : Thread_intf.S) = struct
         in
         let this_val_branch =
           let* () = add_breadcrumb (Int32.to_int i) in
-          let+ () = add_pc this_value_cond in
+          let* () = add_pc this_value_cond in
+          let+ () = modify_thread Thread.mark_pending_pc_done in
```

Comment:
So this is adding something to the pending PC and then emptying the pending PC immediately?